scripts/remote-dev.js

```diff
@@ -0,0 +1,126 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
+const path = require('path');
+const fs = require('fs');
+const crypto = require('crypto');
+const execSync = require('child_process').execSync;
+const helpers = require('../config/helpers');
+
+// Init some consts
+const configDir = path.join(__dirname, '..', 'config');
+const oauthclientFile = path.join(configDir, 'remote.oauthclient.templ.yaml');
+const remoteConfigFile = path.join(configDir, 'config.dev.json');
+const oauthClientTemplateFile = path.join(configDir, 'remote.oauthclient.templ.yaml');
+
+// Validations
+if (!fs.existsSync(remoteConfigFile)) {
+  console.error(`ERROR: Remote config file ${remoteConfigFile} is missing`);
+  console.error(`You could should copy the example to that location and edit with desired config`);
+  process.exit(1);
+}
+
+try {
+  execSync('hash oc');
+} catch (error) {
+  console.error(error.stdout.toString());
+  process.exit(1);
+}
+
+try {
+  execSync('oc whoami');
+} catch (error) {
+  console.error('ERROR: A problem occurred while trying to log into openshift cluster:');
+  console.error('This script uses the oc cli tool, are you logged in with oc login <cluster>?');
+  process.exit(1);
+}
+
+// Helpers
+
+function setupOAuthClient() {
+  const remoteConfig = JSON.parse(fs.readFileSync(remoteConfigFile));
+  const oauthRedirectUri = `http://localhost:${remoteConfig.devServerPort}/login/callback`;
+
+  const oauthClientName = 'mig-ui';
+  const remoteDevSecret = 'bWlncmF0aW9ucy5vcGVuc2hpZnQuaW8K';
```

Comment:
We can just use `remoteConfig.oauth.clientSecret` or whatever here too instead of duplicating it.